packages/core/src/forms/control-group/control-group.element.ts

```diff
@@ -34,7 +34,7 @@ import {
   defaultFormLayout,
   defaultControlWidth,
   getCurrentMessageStatus,
-} from '../utils/index.js';
+} from '../utils/utils.js';
```

Comment:
Rollup doesnt recommend nested barrel/index files for treeshaking reasons. So we can use `index` as the convention for top level package entrypoints.